GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -382,10 +383,18 @@ void InitCountArtificial(out GitStatusMonitor gitStatusMonitor)
 
                         if (AppSettings.ShowSubmoduleStatus)
                         {
-                            _submoduleStatusProvider.UpdateSubmodulesStatus(Module.WorkingDir, status);
+                            try
```

Comment:
Instead of converting to `async void` we should wrap the call in `ThreadHelper.JoinableTaskFactory.Run` instead:
```cs
ThreadHelper.JoinableTaskFactory.Run(() => 
{
    try
    {
        await TaskScheduler.Default;
        await _submoduleStatusProvider.UpdateSubmodulesStatusAsync(Module.WorkingDir, status);
    }
    catch (GitConfigurationException ex)
    {
        await this.SwitchToMainThreadAsync();
        MessageBox.Show(....);
    }
});
```